src/internal/packager2/deploy.go

```diff
@@ -662,7 +676,10 @@ func processComponentFiles(ctx context.Context, pkgLayout *layout.PackageLayout,
 
 		fileList := []string{}
 		if helpers.IsDir(fileLocation) {
-			files, _ := helpers.RecursiveFileList(fileLocation, nil, false)
+			files, err := helpers.RecursiveFileList(fileLocation, nil, false)
```

Comment:
From what I can tell this can only fail if while getting the recursive file list one of the files is deleted